mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -818,45 +818,61 @@ def __repr__(self):
 
 
 @PIPELINES.register_module()
-class IndoorPointSample(object):
-    """Indoor point sample.
+class PointSample(object):
+    """Point sample.
 
     Sampling data to a certain number.
 
     Args:
-        name (str): Name of the dataset.
         num_points (int): Number of points to be sampled.
+        dist_metric (int, optional): The indicator to the near/far boundary
     """
 
-    def __init__(self, num_points):
+    def __init__(self, num_points, dist_metric=None):
         self.num_points = num_points
-
-    def points_random_sampling(self,
-                               points,
-                               num_samples,
-                               replace=None,
-                               return_choices=False):
+        self.dist_metric = dist_metric
+
+    def _points_random_sampling(self,
+                                points,
+                                num_samples,
+                                dist_metric=None,
+                                replace=None,
+                                return_choices=False):
         """Points random sampling.
 
         Sample points to a certain number.
 
         Args:
             points (np.ndarray | :obj:`BasePoints`): 3D Points.
             num_samples (int): Number of samples to be sampled.
-            replace (bool): Whether the sample is with or without replacement.
-            Defaults to None.
-            return_choices (bool): Whether return choice. Defaults to False.
-
+            dist_metric (int, optional): Indicator to the near/far boundary.
```

Comment:
sample_range might be more clear, indicating the range where the points will be sampled.